src/rgw/rgw_op.cc

```diff
@@ -688,6 +689,93 @@ int RGWOp::verify_op_mask()
   return 0;
 }
 
+int RGWGetObjTags::verify_permission()
+{
+  if (!verify_object_permission(s, RGW_PERM_READ))
+    return -EACCES;
+
```

Comment:
This looks like it conflicts with the new bucket policy stuff, but should be easily adaptable